front/src/model/model.js

```diff
@@ -0,0 +1,107 @@
+import { EventEmitter } from "events";
+import RestClient from "../rest/RestClient";
+
+
+const client = new RestClient("serban", "password");
```

Comment:
You should get the username / password from your login UI and not have them hardcoded.